server/README.md

```diff
@@ -66,6 +66,23 @@ Generate a sample SQLite metadata DB:
 docker run --rm -e DB_DIR=/io -v$(pwd):/io --entrypoint python3 athenian/api /server/tests/gen_sqlite_db.py
 ``` 
 
+ ## Running the api locally
+
+Alternatively, you can locally build and run a docker image of the API:
+
+```
+# Build the API image
+docker build -t athenian/api .
+
+# Generate metadata db with fixtures
+docker run --rm -e DB_DIR=/io -v$(pwd):/io --entrypoint python3 athenian/api /server/tests/gen_sqlite_db.py
+
+# Run the API container
+docker run --rm -p 8080:8080 -v$(pwd):/io athenian/api --ui --metadata-db=sqlite:///io/db.sqlite --state-db=sqlite://
```

Comment:
I think that a `run-api` target in the makefile would make things easier:
```
.PHONY: run-api
run-api: docker-build
    docker run --rm -e DB_DIR=/io -v$(pwd):/io --entrypoint python3 athenian/api /server/tests/gen_sqlite_db.py
    docker run --rm -p 8080:8080 -v$(pwd):/io athenian/api --ui --metadata-db=sqlite:///io/db.sqlite --state-db=sqlite://
```

I don't know if the creation of the db fixtures is idempotent, but something along those lines.

Reply:
I writes a new file every time.

Reply:
It could rely on a pre-condition, to avoid creating the fixtures if they already exist:
```makefile
IMAGE ?= athenian/api:dev

db.sqlite:
	docker run --rm -e DB_DIR=/io -v$(PWD):/io --entrypoint python3 $(IMAGE) /server/tests/gen_sqlite_db.py

.PHONY: run-api
run-api: db.sqlite
	docker run --rm -p 8080:8080 -v$(PWD):/io $(IMAGE) --ui --metadata-db=sqlite:///io/db.sqlite --state-db=sqlite://
```

Also, it would separate the building of the image from the running, because if the users already have a production one, they could avoid the (slow) build by doing:
```shell
$ IMAGE=gcr.io/athenian-1/api:master-8e0dfb1 make run-api
```

Shall I add this to this PR @vmarkovtsev @se7entyse7en ?

Reply:
I am fine if this is a separate PR or not, it's up to you.